src/components/SuccessContent.tsx

```diff
@@ -35,13 +51,35 @@ const SuccessContent = ({ title, email }) => {
       <Typography variant="body1">
         {t(AUTH.SIGN_IN_SUCCESS_EMAIL_PROBLEM)}
       </Typography>
+      <br />
+      <Button
+        variant="outlined"
+        color="primary"
+        onClick={onClickResendEmail}
+        id={RESEND_EMAIL_BUTTON_ID}
+        disabled={clicked}
+      >
+        {t(AUTH.RESEND_EMAIL_BUTTON)}
+      </Button>
+      <br />
+      <br />
+      <Button
+        variant="outlined"
+        color="primary"
+        id={BACK_BUTTON_ID}
+        onClick={handleBackButtonClick}
+      >
+        {t(AUTH.BACK_BUTTON)}
+      </Button>
```

Comment:
Layout wise, I think it would be prettier if:
- back button is not outlined (I would prefer `text`)
- align both button on the same line

Reply:
It looks like this now instead. I don't know how much space it should be between the buttons?

<img width="1364" alt="Screenshot 2023-03-14 at 12 53 20" src="https://user-images.githubusercontent.com/56155987/224993193-1ebd3723-6c9a-42e2-af99-91fbbb5a9c4e.png">

Reply:
You can put them in a horizontal `<Stack direction="row" spacing={1}>`

Reply:
I would put "back" on the left, otherwise all good!

Reply:
It's fixed now 

<img width="1305" alt="Screenshot 2023-03-14 at 14 58 53" src="https://user-images.githubusercontent.com/56155987/225024634-84e6f05f-a006-400e-9839-c2a0faac152a.png">